CHANGELOG.md

```diff
@@ -1,23 +1,45 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Version 1.1.0 (15NOV21)
+
+* **Monitoring**
+  * [CHANGE] Most monitoring component versions have been updated
+    * kube-prometheus-stack Helm chart upgraded from 15.0.0 to 19.0.3
+    * Prometheus Operatator upgraded from 0.47.0 to 0.51.2
+    * Prometheus upgraded from 2.26.1 to 2.30.3
+    * Alertmanager upgraded from 0.21.0 to 0.23.0
+    * Grafana upgraded from 7.5.4 to 8.2.1
+    * Node Exporter upgraded from 1.0.1 to 1.2.2
+    * kube-state-metrics upgraded from 1.9.8 to 2.2.1
+  * [FIX] Several dashboards were fixed to adjust to the kube-state-metrics 2.x metrics
+  * [FIX] The KubeHpaMaxedOut alert has been patched to not fire when max instances == current instances == 1
+
+* **Logging**
+  * [CHANGE] Open Distro for Elasticsearch (i.e. Elasticsearch and Kibana) upgraded to version 1.13.2. This includes significant changes to Kibana user-interface, see <https://github.com/sassoftware/viya4-monitoring-kubernetes/tree/master/logging#important-information-about-kibana-in-the-new-release> for details.
+
```

Comment:
Link formatting needs tweaks.  Should look like this one (pulled from elsewhere in the doc): [Deploying Monitoring on OpenShift](https://github.com/sassoftware/viya4-monitoring-kubernetes/blob/master/monitoring/OpenShift.md)

Basically: [ + human readable text + ] + ( + somelink + )